libs/utils/utils.js

```diff
@@ -1293,6 +1293,12 @@ async function loadPostLCP(config) {
     import('../features/personalization/personalization.js')
       .then(({ addMepAnalytics }) => addMepAnalytics(config, header));
   }
+  // load privacy if quick-link is present
+  if (document.querySelector('.quick-link')) {
```

Comment:
If we feel like from a performance perspective this needs to live here, that's fine... but having block specific code in utils is a pretty big anti-pattern.
Maybe you can put it in `preloadBlockResources` which already serves as a 'function to optimize performance within utils'?

Otherwise this ideally would live in the branch links itself.

Reply:
Hi @mokimo 
Thanks for your insights on this. However, placing this in `preloadBlockResources` could impact the LCP more, since it executes before the block is even rendered. Using `loadPostLCP`, on the other hand, ensures the code runs after LCP and is likely to have the least impact on performance.